### PR TITLE
Add template option

### PR DESCRIPTION
**Pull request by alaibe**
screenshot at sep 18 10-22-15

### CODE REVIEW

File: source/docs/create_project.md [hunk 1]
```diff
@@ -28,3 +28,30 @@ $ cd &lt;YourDAppName&gt;
 </code></pre>
 
 This will create a simple project with all components disabled except the blockchain/contracts component.
+
+
+### Creating a project from a template
+
+If you want to start from a pre-existing template, then you can use the `--template` option:
```
Comment (emizzle) [inline source/docs/create_project.md]: Are there any other pre-existing templates besides `angular`?

Comment (alaibe) [reply source/docs/create_project.md]: There is another story about adding them, once added maybe we can add a link to it?

File: source/docs/create_project.md [hunk 2]
```diff
@@ -28,3 +28,30 @@ $ cd &lt;YourDAppName&gt;
...
+
+<pre><code class="shell">$ embark new &lt;YourDAppName&gt; --template angular
+$ cd &lt;YourDAppName&gt;
+</code></pre>
+
+##### Using template with URL
```
Comment (michaelsbradleyjr) [inline source/docs/create_project.md]: I think the template downloader only supports GitHub repositories at present, so maybe change "URL" to "GitHub URL" and on L47 maybe change "GitHub" to "GitHub shortcut" (or shorthand, something like that).

Comment (alaibe) [reply source/docs/create_project.md]: Thx updated

**Pull request merged**